app.js

```diff
@@ -20,14 +20,25 @@ Project.prototype.toHtml = function(){
   $newprojectData.removeClass('template');
```

Comment:
**TODO FOR LOST POINTS**

This is a top level comment that your app.js file should be inside a ``scripts`` folder with other js files.